server/src/controllers/Auth/login.js

```diff
@@ -10,7 +10,7 @@ const login = async (request, response) => {
     const user = rows[0];
```

Comment:
Here are X reasons why this file is completely wrong from start to finish:
1. A UI = User interface, which is the collection of buttons and inputs and fields in the view of you, as a user.
An API = Application Program Interface, think of yourself as a program, what would your interface be? If you were a performant, generic program, your interface will be a collection of short messages and codes.
Saying all this because, even if your audience was targeting Arabs, your API should never throw, return, respond with any Arabic text. A server's task is to supply a result, the Client's responsibility is to translate that result into understandable text.
2. You are throwing multiple errors, and then catching them all, and responding with something completely different, why? Why specify a message if you're not using it?
3. Why is every controller you have handling 500 errors? 
4.
```
if (!validatedPassword) {
      throw
...
if (validatedPassword) 
...
else throw
```
Are you just writing code? Without considering any logic?